src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -520,6 +520,9 @@
         - name: --aks-custom-headers
           type: string
           short-summary: Send custom headers. When specified, format should be Key1=Value1,Key2=Value2
+        - name: --max-surge
+          type: string
+          short-summary: extra nodes used to speed upgrade. When specified, number or percent needed, eg. 5 or 33%
```

Comment:
Nit: please capitalize "extra" for consistency. Also "Extra nodes used to speed up an upgrade operation. When specified, it represents the percentage of additional nodes that will be provisioned (e.g. 5% or 33%)

Reply:
Improved but not always a percent. Can just be a raw number too. Just like a kubernetes deployment's max_surge